go/test/endtoend/recovery/unshardedrecovery/recovery.go

```diff
@@ -185,100 +185,100 @@ func TestMainImpl(m *testing.M) {
 }
 
 // TestRecoveryImpl does following
-// - create a shard with primary and replica1 only
-// - run InitShardPrimary
-// - insert some data
-// - take a backup
-// - insert more data on the primary
-// - take another backup
-// - create a recovery keyspace after first backup
-// - bring up tablet_replica2 in the new keyspace
-// - check that new tablet does not have data created after backup1
-// - create second recovery keyspace after second backup
-// - bring up tablet_replica3 in second keyspace
-// - check that new tablet has data created after backup1 but not data created after backup2
-// - check that vtgate queries work correctly
+// 1. create a shard with primary and replica1 only
+//   - run InitShardPrimary
+//   - insert some data
+//
+// 2. take a backup
+// 3.create a recovery keyspace after first backup
+//   - bring up tablet_replica2 in the new keyspace
+//   - check that new tablet has data from backup1
+//
+// 4. insert more data on the primary
+// 5. take another backup
+// 6. create a recovery keyspace after second backup
+//   - bring up tablet_replica3 in the new keyspace
+//   - check that new tablet has data from backup2
+//
+// 7. insert more data on the primary
+// 8. take another backup
+// 9. create a recovery keyspace after second backup again
+//   - bring up tablet_replica4 in the new keyspace
+//   - check that new tablet has data from backup2 but not backup3
```

Comment:
Related to my previous comment, I don't see where we bring up replica4. And the second part currently seems to be checking that replica3 has data from backup1 but not backup2. No?

Reply:
ahh yes sorry, let me re-do this